semgrep-core/data/basic_tainting.yml

```diff
@@ -1,5 +1,6 @@
 rules:
   - id: classic
+    mode: track
```

Comment:
FWIW, I'd probably `s/track/taint` here for specificity

Reply:
what does s/track/taint mean? something like track-mode: True where track-mode: False is the default?

Reply:
Oohh just swap the words 🤦‍♀️  yes will do